docs/index.html

```diff
@@ -1 +1,1852 @@
-zigggg ziggggggg
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Good but maybe next time we put the index.html change in an other PR ?

Reply:
what do you mean ?
every time you do a PR you could generate this html file again so that our hosted website is always up to date ?

Reply:
No, it's the inverse, we create a PR only for updating the index